.github/workflows/tests.yml

```diff
@@ -227,3 +227,72 @@ jobs:
           path: ${{ env.COVERAGE_DIR }}/cobertura.xml
           include-hidden-files: true
 
+  tics:
+    name: Tiobe TICS
+    runs-on: ubuntu-24.04
+    env:
+      COVERAGE_DIR: ${{ github.workspace }}/.coverage
+      SRC_DIR: ${{ github.workspace }}/microovn/
+    needs:
+      - generate-coverage
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up Go
+        uses: actions/setup-go@v5
+        with:
+          go-version-file: ${{ env.SRC_DIR }}/go.mod
+
+      - name: Install dqlite
+        run: |
+          git clone --branch v1.17.1 --single-branch https://github.com/canonical/dqlite /tmp/dqlite
```

Comment:
The tag used here will update regularly and be different on a per branch basis.

We really do not want two places to maintain this, could you think of a way of retrieving this information from the `snapcraft.yaml`?